src/components/sbb-radio-button-group/sbb-radio-button-group.tsx

```diff
@@ -79,43 +78,43 @@ export class SbbRadioButtonGroup extends LitElement {
    * The value of the radio group.
    */
   @property()
-  public get value(): any | null | null {
+  public get value(): any | null {
     return this._value;
   }
-  public set value(value: any | null | null) {
+  public set value(value: any | null) {
     const oldValue = this._value;
     this._value = value;
     this._valueChanged(this._value);
     this.requestUpdate('value', oldValue);
   }
-  private _value: any | null | null = null;
+  private _value: any | null = null;
 
   /**
    * Size variant, either m or s.
    */
   @property()
-  public get size(): InterfaceSbbRadioButtonGroupAttributes['size'] {
+  public get size(): 's' | 'm' {
```

Comment:
Shouldn't it take the size of the radio button?

Reply:
Yes, also changed in checkbox/checkbox-group